documentation/public/user-guide/09-appendix/01-slang-versioning-policy/index.md

```diff
@@ -0,0 +1,39 @@
+# 9.1. Slang Versioning Policy
+
+Slang is not tied to a specific Solidity version, and can be used to analyze multiple Solidity versions with the same API.
+At the same time, Slang's own API needs to be consistent and stable as new versions of Solidity are released.
+
+The NPM package uses [Semantic Versioning](https://semver.org/) and [CHANGELOG.md](https://github.com/NomicFoundation/slang/blob/main/CHANGELOG.md)
+to communicate such changes in the form of `{MAJOR}.{MINOR}.{PATCH}` versions, to report breaking changes, new features, and bug fixes (respectively).
+But it is not not really designed to report two (often orthogonal) kinds of changes: changes in Slang's APIs, and changes in the Solidity language itself.
+
+Therefore, it is important to define and follow a clear versioning policy for Slang, that will make this distinction clear.
+Each changelog entry will contain a link to the pull request that introduced the change. The pull request will contain a description
+of the change, its impact on our APIs, the Solidity language, and any potential impact on the users.
+
+## Major Version Updates
+
+These are `{MAJOR}.0.0` releases, and they are intended to represent breaking changes in Slang's APIs.
+For example, when an existing API is removed, or replaced with a different one, and will often require
+manual intervention from the user to update their code.
+
+These releases are only done when absolutely necessary, and are not expected to happen often.
+They are often accompanied by detailed announcements, explaining the motivation behind it.
+
+## Minor Version Updates
```

Comment:
Maybe this section could include a comment explaining that changes in the CST and the like are technically breaking changes, but due to changes in the language, and not Slang's own APIs.

Also, it could recommend installing with `~` instead of `^` for those that want total stability.

Reply:
Done.